test/compilable/cppmangle.d

```diff
@@ -768,3 +768,20 @@ version(Windows)
       "??$test19043b@U?$test19043@$$CBD@@@@YAXU?$test19043@$$CBD@@@Z");
 }
 
+/**************************************/
+// https://issues.dlang.org/show_bug.cgi?id=19278
+// extern(C++, "name") doesn't accept expressions
+
+extern(C++, "hello" ~ "world")
+{
+    void test19278();
+}
+extern(C++, "hello" ~ "::" ~ "world")
+{
+    void test19278_2();
+}
+version(Win64)
+{
+    static assert(test19278.mangleof == "?test19278@helloworld@@YAXXZ");
+    static assert(test19278.mangleof == "?test19278_2@world@hello@@YAXXZ");
+}
```

Comment:
Did you implement support for `extern (C++, (foo))`?

Reply:
yeah i'm trying to work that out... i think the moment in semantic where i inserted my code is too early to perform symbol lookups :/
what's the proper earliest moment in compilation where ctfe is able to resolve symbols? I think I need to move this code to there...